### PR TITLE
Remove opensearch code

### PR DESCRIPTION
**Pull request by olaughter**
Having moved to using Vespa for search we can get rid of the Opensearch code as well as corresponding tests and documentation. If there is any more that we miss from this we can follow up with further cleanup but I think this is all of it

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [x] Removing Opensearch

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [x] The PR represents a single feature (small driveby fixes are also ok)
- [x] The PR includes tests that are sufficient for the level of risk
- [x] The code is sufficiently commented, particularly in hard-to-understand areas
- [x] Any required documentation updates have been made
- [x] Any TODOs added are captured in future tickets
- [x] No FIXMEs remain

### CODE REVIEW

File: .env.local
Comment (diversemix) [inline .env.local:1]: Ooo - not suggesting we change this as part of this PR. But shouln't this file be something like `.env.local.example` ?

Comment (olaughter) [reply .env.local:1]: Probably! Although I'm not sure why we need so many either... maybe we can delete one

File: app/core/sensitive_query_terms.tsv
Comment (diversemix) [inline app/core/sensitive_query_terms.tsv:1]: Ooo - what are we dong with sensitive terms in Vespa?

Comment (olaughter) [reply app/core/sensitive_query_terms.tsv:1]: We do this in the data access library now: https://github.com/climatepolicyradar/data-access/blob/main/src/cpr_data_access/search_adaptors.py#L85

Comment (diversemix) [reply app/core/sensitive_query_terms.tsv:1]: Now here https://github.com/climatepolicyradar/data-access/blob/main/src/cpr_data_access/search_adaptors.py#L85

**Review by diversemix — APPROVED** [review]
Looks good to me! Let's get this on staging :raised_hands:

**Pull request merged**